### ISSUE TITLE
E265 is only partially enforced, failing to fix multiple spaces after #

### ISSUE THREAD

**Issue by damonlynch** [astral-sh/ruff#22906]
### Summary

Currently this is fixed, because there is no space:
`#Comment with weird spacing`

But this is not, because there is already at least one space:
`#     Another comment with weird spacing`

The relevant [Flake8 rule](https://www.flake8rules.com/rules/E265.html) states "Block comments should have one space before the pound sign (#) and the comment itself."

Thank you, Astral. You all are doing amazing work. 🙏

**Comment by damonlynch** [comment]
When filing this issue, I did not think of the situation where code itself is commented. My mistake. 

Commented code is an obvious pitfall to rigorously enforcing this rule. To detect the difference between code and a regular comment, this project may be helpful: https://github.com/PyCQA/eradicate

**Comment by ntBre** [comment]
Thanks for the report and for the kind words! Yes, based on the rule name and testing against the pycodestyle linter itself, I believe this is working as intended. The rule enforces at least one space rather than exactly one space.

We do also have an eradicate  rule: [commented-out-code (ERA001)](https://docs.astral.sh/ruff/rules/commented-out-code/#commented-out-code-era001).

**Comment by damonlynch** [comment]
Thanks for the very quick response. Please allow me to let you know what prompted me to open the issue.

As I'm sure you know, the [REUSE Specification](https://reuse.software/) specifies a header format to identify what license is a file licensed under, and who owns the copyright. 

[PyCharm's Copyright plugin](https://www.jetbrains.com/help/pycharm/copyright.html) can auto-generate and update this header — a very handy feature.

Unfortunately for [some years now the plugin](https://youtrack.jetbrains.com/issue/PY-60543/Two-Spaces-After-Comment-Character-In-Generated-Copyright-Headers-for-Python-Violates-PEP-8) has been inserting two spaces between the leading # sign and the comment body. Pretty annoying!

Is it within Ruff's scope to develop a new rule to check the format of REUSE Spec headers?

Incidentally, the first issue I filed against Ruff was actually allowing these headers to sometimes exceed the project line length without word wrapping, which Charlie implemented.

**Comment by ntBre** [comment]
Oh interesting, thank you for the additional context! Maybe it would make sense to enforce exactly one space in `E265` after all, or at least make that a configurable option. That sounds a bit better to me than a separate rule, but I'm curious what others think.

**Comment by amyreese** [comment]
My concern with enforcing a single space in E265 is that it would fire every time someone wants to format their comments in some special way like:

```py
# List of things:
#  - thing 1
```
